pkg/composer/group.go

```diff
@@ -116,6 +123,35 @@ func (c *composer) ComposeGroup(group brevity.Group) NaturalLanguageResponse {
 	}
 }
 
+// ComposeMergedWithGroup is a short form of describing a group for use in merge calls.
+func (c *composer) ComposeMergedWithGroup(group brevity.Group) NaturalLanguageResponse {
+	var speech, subtitle strings.Builder
+	if group.Contacts() > 1 {
+		contacts := c.ComposeContacts(group.Contacts())
+		speech.WriteString(contacts.Speech + ", ")
+		subtitle.WriteString(contacts.Subtitle + ", ")
+	}
+	if len(group.Platforms()) > 0 {
+		platforms := strings.Join(group.Platforms(), ", ")
+		speech.WriteString(platforms)
+		subtitle.WriteString(platforms)
+	}
```

Comment:
> if its included, should be configurable, since depending on the scenario being roleplayed, types may not be available anyway.

My longer term plan for this is now to filter this upstream in the ACMI data. The existing code path will then not populate group.Platforms() and line 134 will eval to false and skip the block.

> eventually when tactical pictures are implemented, none of this (including contacts) will be needed since itll all be contained in a group label. see example below

Maybe, but my preference is to write code for the environment you're working with today and rewrite it for the future. Programmers are very bad at predicting the future.

> they should be able to VID the threat very soon if they didn't already know it from other information (FLIR, NCTR, RWR, previous calls)

> during a merge, AIC/ABM has the lowest comm priority, so the longer this comm is, the more PRI is being jammed up.

These are good points, especially the more complex the furball becomes the more redundant info is gonna get included. I'll drop this block from the next revision.

Reply:
Dropped from merged call.